packages/api-server/src/tests/authenticator.test.ts

```diff
@@ -0,0 +1,34 @@
+import * as jwt from 'jsonwebtoken';
+import WebSocket from 'ws';
+import auhthenticator from '../authenticator';
+import WebSocketConnect, { WebSocketMiddleware } from '../websocket-connect';
+
```

Comment:
Should we add tests expecting errors on the jwt verification and on the token type checking?

Reply:
good point! I will add these.